run-ib-igprof

```diff
@@ -8,8 +8,8 @@ if [ "X$EVENTS" = "X" ] ; then EVENTS=1000; fi
 for prof in ${PROFILES} ; do
   mkdir -p $WORKSPACE/igprof/${prof}
   cd $WORKSPACE/igprof/${prof}
-  runTheMatrix.py $WORKFLOWS --ibeos --command "-n $EVENTS --profile $prof" 2>&1 | tee runTheMatrix.log
-  for f in $(find . -name '*.gz' -type f) ; do
+  runTheMatrix.py $WORKFLOWS --ibeos --command "-n $EVENTS --profile $prof --nThreads=1" 2>&1 | tee runTheMatrix.log
```

Comment:
the default threads is set to 1 9 https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/scripts/runTheMatrix.py#L126 ) so why you need to explicitly set it here?

Reply:
I thought I saw 4 threads in the log somewhere. Maybe that was ROOT.